<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>238</width>
     <height>305</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="line_mmb">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="chb_dag">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>same dag nodes</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="line_lmb">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="cmb_lmb">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>mel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>python</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QCheckBox" name="chb_non_dag">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>same non dag nodes</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QCheckBox" name="chb_radial">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>radial menu for presets</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>HUD section</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>MMB (tools) default marking menu</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="btn_save">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QCheckBox" name="chb_hud">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>heads-up display for presets</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="cmb_mmb">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>mel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>python</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QSpinBox" name="spin_hud_s">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QSpinBox" name="spin_hud_b">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>HUD block</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>LMB (selected) default marking menu</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QCheckBox" name="chb_cache">
      <property name="text">
       <string>cache mel marking menus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
